script.js

```diff
@@ -38,3 +38,89 @@ document.addEventListener("DOMContentLoaded", function () {
     closeButton.style.display = "none";
   }
 });
+
+document.addEventListener("DOMContentLoaded", () => {
+  function openModal(modal) {
+    if (modal == null) return;
+    modal.classList.add("active");
+    overlay.classList.add("active");
+  }
+
+  function closeModal(modal) {
+    if (modal == null) return;
+    modal.classList.remove("active");
+    overlay.classList.remove("active");
+  }
+
+  const openModalButtons = document.querySelectorAll(
+    '[data-modal-target="#modal"]'
+  );
+  const closeModalButtons = document.querySelectorAll("[data-btn-close]");
```

Comment:
* As stated by the [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md#:~:text=You%20need%20to%20use%20a%20JavaScript%20array%20to%20store%20all%20of%20the%20information%20for%20all%20projects.) You need to use a JavaScript array to store all of the information for all projects, currently you have hardcoded all projects info, kindly follow the requirement 
* ![image](https://github.com/Eccb7/Portfolio2.0/assets/52497246/fc580219-b391-4ff4-b9d9-21690912dfbc)